libs/libencrypt/src/encryption.cpp

```diff
@@ -116,47 +181,52 @@ std::string Encryption::base64_encode(const std::string& input) {
  */
 bool Encryption::encryptFile(const std::string& publicKeyFile, std::string& filePath ) {
 
-    // Load public key
-    RSA* publicKey = loadPublicKey(publicKeyFile);
+    OpenSSL_add_all_algorithms();
+    ERR_load_crypto_strings();
+    unsigned char sessionKey[16];
```

Comment:
can we avoid using constant number here ("16") directly?  we can use a meaningful constant variable to represent it.